browser/ipfs/test/BUILD.gn

```diff
@@ -21,6 +21,7 @@ source_set("unittests") {
     "//brave/browser",
     "//brave/browser/net:net",
     "//brave/browser/tor",
+    "//brave/components/decentralized_dns",
```

Comment:
Add `//brave/components/decentralized_dns/buildflags` as a dep here and move this one to be guarded with decentralized_dns_enabled.